core/src/test/java/nl/nn/adapterframework/jdbc/JdbcSenderTestBase.java

```diff
@@ -25,9 +25,9 @@
 	public void setUp() throws Exception {
 		session = new PipeLineSession();
 		String messageId = "testmessageac13ecb1--30fe9225_16caa708707_-7fb1";
```

Comment:
Misschien dat we hier een global ding voor kunnen maken?

Reply:
gedaan